recipes/libexif/meta.yaml

```diff
@@ -0,0 +1,33 @@
+{% set name = "libexif" %}
+{% set version = "0.6.21" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://sourceforge.net/projects/{{name}}/files/{{name}}/{{version}}/{{name}}-{{version}}.tar.gz
+  sha256: edb7eb13664cf950a6edd132b75e99afe61c5effe2f16494e6d27bc404b287bf
+
+build:
+  number: 0
+  skip: True  # [win]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+  host:
+  run:
```

Comment:
Could you please remove those empty sections?

Reply:
Sure.